modules/op-release-notes-1-10.adoc

```diff
@@ -166,3 +166,13 @@ With this update, {pipelines-title} General Availability (GA) 1.10.1 is availabl
 * Before this update, when a `PipelineRun` object could not be created, the error received from the Tekton controller was not reported to the user. With this update, {pac} reports the error messages to the GitHub interface so that users can troubleshoot the errors. {pac} also reports the errors that occurred during pipeline execution.
 * With this update, {pac} does not echo a secret to the GitHub checks interface when it failed to create the secret on the {product-title} cluster because of an infrastructure issue.
 * This update removes the deprecated APIs that are no longer in use from {pipelines-title}.
+
+[id="release-notes-1-10-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.2
+
+With this update, {pipelines-title} General Availability (GA) 1.10.2 is available on {product-title} 4.11 and later versions.
+
+[id="fixed-issues-1-10-2_{context}"]
+=== Fixed issues
+
+Before this update, Tekton Operator had a bug that prevented the user from setting the value of the `enable-api-fields` flag to `beta`. This update fixes the issue and you can use the value `beta` for configuring the `enable-api-fields` flag in the `TektonConfig` CR.
```

Comment:
Can we change this to as follows:

Before this update, an issue in the Tekton Operator prevented the user from setting the value of the `enable-api-fields` flag to `beta`. This update fixes the issue. Now you can set the value to `beta` and configure the `enable-api-fields` flag in the `TektonConfig` CR.

Reply:
Incorporated